packages/truffle-decode-utils/src/contexts.ts

```diff
@@ -97,7 +96,7 @@ export namespace Contexts {
             debug("signature read: %s", signature);
             //if not, compute it ourselves
             if(signature === undefined) {
-              signature = abiCoder.encodeFunctionSignature(<AbiItem>abiEntry);
+              signature = abiCoder.encodeFunctionSignature(<any>abiEntry);
```

Comment:
Doing this since #2214 demonstrates the same `AbiItem` failure as on this branch, and I think this shouldn't impact the web3 release validation.